EsperBenchmark/TimeWindowBenchmark/src/main/java/Main.java

```diff
@@ -0,0 +1,158 @@
+import com.espertech.esper.common.client.EPCompiled;
```

Comment:
Lets have a meaningful class name than Main.java

Reply:
Changed the class name to Benchmark